nestkernel/sp_manager.cpp

```diff
@@ -417,7 +422,7 @@ SPManager::update_structural_plasticity( SPBuilder* sp_builder )
  * @param post_n number of available synaptic elements in the post node
  * @param sp_conn_builder structural plasticity connection builder to use
  */
-void
+bool
```

Comment:
Should probably add a `@return` entry after the `@param` entries above, giving a short description of the return value.